.github/workflows/bump-image-version.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  repository_dispatch:
+    types: [bump_image_version]
+  workflow_dispatch:
+    inputs:
+      component:
+        description: The helm service component to update
+        type: choice
+        options:
+          - api
+          - frontend
+          - proxy
+          - recording
+          - sockets
+        required: true
+      component_version:
+        description: The new version of the component
+        type: string
+        required: true
+jobs:
+  bump-image-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Resolve inputs
+        run: |
+          echo "INPUT_COMPONENT=${{ inputs.component || github.event.client_payload.component }}" >> $GITHUB_ENV
+          echo "INPUT_COMPONENT_VERSION=${{ inputs.component_version || github.event.client_payload.component_version }}" >> $GITHUB_ENV
+      - name: Checkout source code
+        uses: actions/checkout@v3
+      - name: Update deployment version
+        run: sed -Ei 's/(image:.*-enterprise):[0-9.]+/\1:${{ env.INPUT_COMPONENT_VERSION }}/g' templates/${{ env.INPUT_COMPONENT }}-${{ env.INPUT_COMPONENT == 'sockets' && 'statefulset' || 'deployment' }}.yaml
+      - uses: EndBug/add-and-commit@v9
```

Comment:
I'd prefer to avoid third party (except from very trusted sources like GitHub) workflows if possible due to the security considerations. This seems like something that should be easy to do through GitHub scripts

Reply:
Fixed.